mandatory/2-function-creation.js

```diff
@@ -16,7 +18,9 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  let valid = num > 100 ? 100 : parseFloat(num.toFixed(2));
```

Comment:
This is an interesting approach. I think your ternary is sensible, effectively defaulting to 100 if the number is too big, even though I don't think you covered this syntax yet in class.